modules/text_server_adv/text_server_adv.cpp

```diff
@@ -1044,7 +1044,7 @@ _FORCE_INLINE_ TextServerAdvanced::FontGlyph TextServerAdvanced::rasterize_msdf(
 			if (RenderingServer::get_singleton() != nullptr) {
 				Ref<Image> img;
 				img.instantiate();
-				img->create_from_data(tex.texture_w, tex.texture_h, 0, Image::FORMAT_RGBA8, tex.imgdata);
+				img->create_from_data(tex.texture_w, tex.texture_h, false, Image::FORMAT_RGBA8, tex.imgdata);
```

Comment:
The same changes also relevant for `text_server_fb` (this part of code is almost identical).

Reply:
Thanks, changed.